ingredients.css

```diff
@@ -0,0 +1,163 @@
+.yellow-background {
+  background-image: url("https://cdn.glitch.com/1fe73c9b-2068-4454-85a1-ac9b67eb3ef5%2Fmobile%20yellow.png?v=1604381029665 ");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: 100%;
+  height: auto;
+}
+h2 {
+  font-family: "Josefin Sans", sans-serif;
+  color: white;
+}
+
+.dough {
+  background: #F5F5F5;
+  border-radius: 30px;
+  width: 290px;
+  padding: 10px;
+  box-shadow: 0px 5px 6px #D5C48C;
+}
+
+.sauce {
+  background: #F5F5F5;
+  border-radius: 30px;
+  width: 290px;
+  padding: 10px;
+  box-shadow: 0px 5px 6px #D5C48C;
+}
+
+.cheese {
+  background: #F5F5F5;
+  border-radius: 30px;
+  width: 290px;
+  padding: 10px;
+  box-shadow: 0px 5px 6px #D5C48C;
+}
+
+.meats {
+  background: #F5F5F5;
+  border-radius: 30px;
+  width: 290px;
+  padding: 10px;
+  box-shadow: 0px 5px 6px #D5C48C;
+}
```

Comment:
I see a lot of repeated CSS here. Could you make one class that shares all these styles instead, and use that class multiple times in your HTML?